src/socket.js

```diff
@@ -0,0 +1,57 @@
+let DataReceiver = {
+  callbacks: new Map(),
+  onDataReceived: (aSocketId, aData) => {
+    if (!DataReceiver.callbacks.has(aSocketId)) {
+      return;
```

Comment:
This also seems like something that shouldn't happen. Should we log an error here?

Reply:
This line exists on principle; because in theory the WEE API could have multiple consumers and the event might be related to a socket that this DataReceiver is not tracking.